src/Tests/display.test.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import MyToDo from '../app/modules/display.js';
+import List from '../app/modules/class.js';
+
+const newTask = new MyToDo('description', false, 1);
+const todoList = [];
+
+describe('add a todo list', () => {
+  test('add a single task to the list', () => {
+    document.body.innerHTML = `
+            <label for="add" class="add"></label>
+            <input name="add" id="add" placeholder="Add to your list.." />
+            <ul class="listContainer"></ul>`;
+    newTask.addTask(todoList, 'complete assignment');
+    const listContainer = document.querySelectorAll('.listContainer li');
+    expect(listContainer).toHaveLength(1);
+  });
+
+  test('add two tasks to the list', () => {
+    newTask.addTask(todoList, 'Meet friends');
+    newTask.addTask(todoList, 'Update my Resume');
+    const listContainer = document.querySelectorAll('.listContainer li');
+    expect(listContainer).toHaveLength(3);
+  });
+});
+//   test('add task to the list', () => {
+//     list = ('submit my assignment', false, 3);
+//     todoList.push(list);
+//     expect(todoList).toHaveLength(3);
+//   });
+// });
```

Comment:
- [ ] [OPTIONAL] Please kindly remove any code comments as it pollutes the codebase.

Reply:
Thank you for your feedback. I noticed it after I submit but I removed it now.